### PR TITLE
Add .cache class in general_settings/edit.html.erb

### PR DESCRIPTION
**Pull request by jumbosushi**
### What

Add `.cache` class to clear cache card div in `general_settings/edit.html.erb`

### Why

Lately I wanted to create a deface override to clear cache card in `general_settings/edit.html.erb`, but noticed that didn't have a unique class to be used as css selector. I ended up using hacky css selector rule, but this would make it much easier.

### CODE REVIEW

**Comment by MatthewKennedy** [comment]
Personally in this scenario I would use id="cache" or id="cache-settings", each id is intended to be unique to one element in the dom, ideal for targeting something like this, also using an id rather than adding another class clearly separates the bootstrap styling from the id for potential deface override targeting.

File: backend/app/views/spree/admin/general_settings/edit.html.erb [hunk 1]
```diff
@@ -37,7 +37,7 @@
           <%#-------------------------------------------------%>
           <%# Clear cache                                     %>
           <%#-------------------------------------------------%>
-          <div class="card mb-3">
+          <div class="card mb-3 cache">
```
Comment (damianlegawiec) [inline backend/app/views/spree/admin/general_settings/edit.html.erb]:
```suggestion
          <div class="card mb-3" id="general-settings-cache" data-hook="general_settings_cache">
```

**Review by damianlegawiec — CHANGES_REQUESTED** [review]

**Comment by jumbosushi** [comment]
Thanks for the suggestion! Would it be a good idea to make an another PR to add `id` to currency card div as well?

**Comment by damianlegawiec** [comment]
That would be awesome @jumbosushi - thanks!

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**